src/odemis/gui/cont/streams.py

```diff
@@ -2534,6 +2533,39 @@ def _updateMicroscopeStates(self):
                 self._tab_data_model.emState.value = guimodel.STATE_OFF
 
 
+class CryoStreamsController(StreamBarController):
+    """
+    Controls the display of stream panels without affecting the actual streams
+    """
+
+    def _onView(self, view):
+        """ Handle the changing of the focused view """
+        if not view or self.ignore_view:
+            return
+
+        # hide/show the stream panels which are compatible with the view
+        allowed_classes = view.stream_classes
+        for e in self._stream_bar.stream_panels:
+            e.Show(isinstance(e.stream, allowed_classes))
+            ov_streams = self._tab_data_model.overviewStreams
+            static = issubclass(view.stream_classes, StaticStream)
+            # Check view type between Acquired and Overview
```

Comment:
Please explain what do you want to *attain* here. The final goal. I think you have like 3 options: l
* live stream -> live views
* acquired (static) stream related to a feature -> "Acquired" view
* (static) overviews -> "Overview" view

Is that correct?
Then, to distinguish between the 2 views related to StaticStream, it's not possible to only depend on the "stream_classes", so you depend on the class type (which is FeatureOverviewView only for the Overview) + the fact that the stream is in the "overviewStreams". Is that correct? If so, please write all this down in a comment here!